src/gmail.js

```diff
@@ -961,6 +961,26 @@ var Gmail = function(localJQuery) {
             if (pathname && (pathname.endsWith("/i/s") || pathname.endsWith("/i/fd"))) {
                 api.tools.parse_request_payload(params, triggered);
             }
+            if (pathname && (pathname.endsWith("/i/s"))) {
```

Comment:
This whole following section should probably be put inside `api.tools.parse_request_payload()`, like other event's currently parsed.